src/lib/profiles/security/WeaveSecurity.h

```diff
@@ -243,6 +252,45 @@ enum
     kTag_GroupKeySignature_KeyId                = 2,    //  [ unsigned int ] Weave KeyId to be used to generate and verify the signature
     kTag_GroupKeySignature_Signature            = 3,    //  [ byte string ] Signature bytes themselves.
 
+    // ---- Context-specific Tags for GetCertificateRequest Message Structure ----
+    kTag_GetCertReqMsg_ReqType                  = 1,    // [ unsigned int ] Identifies the certificate request type.
+    kTag_GetCertReqMsg_Authorize_PairingToken   = 2,    // [ byte string, optional ] Pairing token from the service, in BER integer encoding.
```

Comment:
The indention used in the comments does not match the reset of the value, and at least to my eyes, is harder to read.

Reply:
updated